runtime/heiko/src/lib.rs

```diff
@@ -958,11 +957,6 @@ type EnsureRootOrHalfCouncil = EnsureOneOf<
     EnsureRoot<AccountId>,
     pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, CouncilCollective>,
 >;
-type EnsureRootOrHalfTechnical = EnsureOneOf<
```

Comment:
Still need this origin for other pallets, like market params change.

Reply:
@kaichaosun yes, in last commit I'd like to use it for `ScheduleOrigin` but finally found that's in pallet-schedule it seems it hardcoded `ensure_root`, and it's the only place that I used EnsureRootOrHalfTechnical, so I remove it for now